sweeper/aws_apprunner_auto_scaling_configuration_version: Can't delete "DefaultConfiguration"

AWS defines a default Auto Scaling configuration for App Runner. This cannot be deleted.

When the sweeper tries, it gets the following error

> error deleting resource: error deleting App Runner AutoScaling Configuration Version (arn:aws:apprunner:us-east-1:*******:autoscalingconfiguration/DefaultConfiguration/1/00000000000000000000000000000001): InvalidRequestException: You can't delete a reserved auto scaling configuration: '%s'.

(Yes, with the literal `%s`)

Do not attempt to sweep `aws_apprunner_auto_scaling_configuration_version`s containing `autoscalingconfiguration/DefaultConfiguration` in the ARN

### Affected Resource

<!--- Please list the affected resources and data sources. --->

* aws_apprunner_auto_scaling_configuration_version

🤔  I wonder if this is because we have dangling apprunner services (at least possibly by the time the sweepers run), given the tests in TC are failing w/
```
testing_new.go:63: Error running post-test destroy, there may be dangling resources: exit status 1
        
        Error: error deleting App Runner Service (arn:aws:apprunner:us-west-2:*******:service/tf-acc-test-8082562876927284703/ace8a25147e74405937b19b99ede81bb): InvalidStateException: Service cannot be deleted in the current state: OPERATION_IN_PROGRESS.
```
The autoscaling config version sweeper has a dependency on the apprunner service sweepers so I would of thought once the service is gone, so is it's default autoscaling config but i'd have to double-check

--- a/aws/resource_aws_apprunner_auto_scaling_configuration_version_test.go
+++ b/aws/resource_aws_apprunner_auto_scaling_configuration_version_test.go
@@ -49,6 +49,13 @@ func testSweepAppRunnerAutoScalingConfigurationVersions(region string) error {
 				continue
 			}
 
+			// Skip DefaultConfigurations as deletion not supported by the AppRunner service
+			// Reference: https://github.com/hashicorp/terraform-provider-aws/issues/19840
+			if aws.StringValue(summaryConfig.AutoScalingConfigurationName) == "DefaultConfiguration" {
+				log.Printf("[INFO] Skipping App Runner AutoScaling Configuration: DefaultConfiguration")
+				continue
+			}
+
 			arn := aws.StringValue(summaryConfig.AutoScalingConfigurationArn)
 
 			log.Printf("[INFO] Deleting App Runner AutoScaling Configuration Version (%s)", arn)

